### ISSUE TITLE
Save JPG or PNG, invalid archive. 

### ISSUE THREAD

**Issue by rankinstudio** [spite/ccapture.js#43]
Archive is either unknown format or damaged. 

Not sure why this is happening. The WEBM and GIF captures work great. Any ideas?

Thanks

**Comment by spite** [comment]
Not really, without more details. What is your setup like, or can you post something online?

**Comment by rankinstudio** [comment]
This is how I'm calling the image capture: 

    function saveJPG(){
        var pngcapturer = new CCapture( { format: 'png' } );
        pngcapturer.start();
        pngcapturer.stop();
        pngcapturer.save();
    }

**Comment by spite** [comment]
Exactly like that? If there are no pngcapturer.capture( canvas ); between .start() and .stop() there are no frames to export.

**Comment by spite** [comment]
Did this get solved?

**Comment by rankinstudio** [comment]
I was never able to get this to work.

**Comment by Geoneer** [comment]
the suggestion of @spite  does not work. I'm struggling with saving just one .png with this library also. Trying to find out how.... hopefully I will get this working.

**Comment by spite** [comment]
@Geoneer ar you sure you're pushing at least a complete frame? Complete means it's rendered and has a valid size (not 0x0)
